cloud-logging/docs/cloudshell-tutorial.md

```diff
@@ -0,0 +1,77 @@
+# Apigee Cloud Logging Tutorial
+
+---
+This sample let you configure an Apigee Proxy that will write custom log entries to GCP Cloud Logging
+
+Let's get started!
+
+---
+
+## Setup environment
+
+Navigate to the 'cloud-logging' drirectory in the Cloud shell.
+
+```sh
+cd cloud-logging
+```
+
+Edit the provided sample `env.sh` file, and set the environment variables there.
+
+Click <walkthrough-editor-open-file filePath="cloud-logging/env.sh">here</walkthrough-editor-open-file> to open the file in the editor
+
+Then, source the `env.sh` file in the Cloud shell.
+
+```sh
+source ./env.sh
+```
+
+Ensure you have an active GCP account selected in the Cloud shell
```

Comment:
Can you move this section to the very top under "Setup environment"
Please refer to https://github.com/GoogleCloudPlatform/apigee-samples/blob/main/deploy-apigee-proxy/docs/cloudshell-tutorial-maven.md#setup-environment

Reply:
moved